packages/x-components/src/x-modules/search/components/__tests__/spellcheck-button.spec.ts

```diff
@@ -35,7 +34,7 @@ function renderSpellcheckButton({
   return {
     wrapper,
     async clickEventButton() {
```

Comment:
Like above, no need to do a `findComponent` or a `localVue.nextTick()`. Rename to just `click` as there is no more event-button.
```ts
    async click() {
      await wrapper.trigger('click');
    }
```